### PR TITLE
Added Hand class -JT

### PR DESCRIPTION
**Pull request by jtamayo0701**
Check out this dope ass Hand class

### CODE REVIEW

File: Hand.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
+import java.io.*; 
+import java.util.*;
+
+public class Hand{
+
+    //fields - cards, point value, bet
+    List <Card> currentCards = new ArrayList<Card>();
```
Comment (jnguyen7410) [inline Hand.java]: nit: List<Card>

Comment (jnguyen7410) [reply Hand.java]: Also, do the init in the constructor

File: Hand.java [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    int pointValue;
+    double bet;
+
+    public Hand()
+    {
+        Card randomCard1 = new Card();
+        Card randomCard2 = new Card();
```
Comment (jnguyen7410) [inline Hand.java]: Remove these.

File: Hand.java [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+        currentCards.add(randomCard1); //Add the first card
+        currentCards.add(randomCard2); //Add the second card
```
Comment (jnguyen7410) [inline Hand.java]: Remove these as well.

File: Hand.java (hunk 2 above)
Comment (jnguyen7410) [reply Hand.java]: Move the init from the field to here.

File: Hand.java [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+    public void addCards()
+    {
+
+    }
+
+    public void revealCards()
+    {
+
+    }
+
```
Comment (jnguyen7410) [inline Hand.java]: Add a function to calculate the hand score

File: Hand.java [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+    {
+        this.bet = bet;
+    }
+
+    public void addCards()
+    {
+
+    }
+
+    public void revealCards()
```
Comment (jnguyen7410) [inline Hand.java]: Overload this function to have an argument with a List of Card

File: Hand.java [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+    {
+        this.pointValue = pointValue;
+    }
+
+    public void setBet(double bet)
+    {
+        this.bet = bet;
+    }
+
+    public void addCards()
```
Comment (jnguyen7410) [inline Hand.java]: There's no arguments. Please add List<Card> as arg

**Review by jurytan — CHANGES_REQUESTED** [review]
Good job so far. Please make the changes I requested.

File: Hand.java (hunk 6 above)
Comment (jtamayo0701) [reply Hand.java]: I have included the List as arg

Comment (jtamayo0701) [reply Hand.java]: Please see Hand Class v2

File: Hand.java (hunk 5 above)
Comment (jtamayo0701) [reply Hand.java]: I have included the List as arg. 
Please see Hand Class v2

File: Hand.java (hunk 4 above)
Comment (jtamayo0701) [reply Hand.java]: Added the calculateHandScore function.

File: Hand.java (hunk 3 above)
Comment (jtamayo0701) [reply Hand.java]: Deleted these.

File: Hand.java (hunk 2 above)
Comment (jtamayo0701) [reply Hand.java]: Deleted it and moved the init within the constructor.

File: Hand.java (hunk 1 above)
Comment (jtamayo0701) [reply Hand.java]: Moved the init to the constructor.

**Comment by jtamayo0701** [comment]
@jnguyen7410 I made the update to my code.

**Comment by jnguyen7410** [comment]
lgtm

**Review by jurytan — APPROVED** [review]

**Pull request merged**